src/main/scala/ai/privado/languageEngine/go/passes/orm/GorpParser.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of Privado OSS.
+ *
+ * Privado is an open source static code analysis tool to discover data flows in the code.
+ * Copyright (C) 2022 Privado, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, contact support@privado.ai
+ *
+ */
+package ai.privado.languageEngine.go.passes.orm
+
+import io.shiftleft.codepropertygraph.generated.nodes.*
+import io.shiftleft.codepropertygraph.generated.{Cpg, EdgeTypes}
+import io.shiftleft.semanticcpg.language.*
+import org.slf4j.LoggerFactory
+
+class GorpParser(cpg: Cpg) extends BaseORMParser(cpg) {
+  val GORP_PARAMETER_TYPE_RULE = "github.com/go-gorp/gorp.*"
+  val ADDRESS_OF_OBJECT_SYMBOL = "*[]"
```

Comment:
is this `*[]` or coding error?

Reply:
not coding error. When object passed as pointer its signature come as `*[]packagename.StructName`

Reply:
Added comment saying so in: https://github.com/Privado-Inc/privado-core/pull/814/commits/1f896170d831e6bfac21ffbb33348b16be85211c#diff-acf64ac8318fa5936d0d13f010c3d12645fc5c510e04ae1428db5659f1ff1ee0